src/main/java/dog/pawbook/model/managedentity/owner/UniqueEntityList.java

```diff
@@ -179,7 +179,7 @@ public void setEntities(List<Pair<Integer, Entity>> entities) {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof UniqueEntityList // instanceof handles nulls
-                        && internalList.equals(((UniqueEntityList) other).internalList));
+                && internalList.equals(((UniqueEntityList) other).internalList));
```

Comment:
this indentation doesn't make sense, since it is a continuation of the bracket of the previous line, the `&&` on this line is not at the same level as the `||` on the previous line.